controllers/sandbox_controller.go

```diff
@@ -80,7 +81,26 @@ func (r *SandboxReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 	}
 
 	oldStatus := sandbox.Status.DeepCopy()
+	var err error
 
+	expired, requeueAfter := r.processSandboxExpiry(sandbox)
+
+	// Check if sandbox has expired
+	if expired {
+		log.Info("Sandbox has expired, deleting pod and service")
```

Comment:
TBD if we want to delete the Service, but let's merge and iterate

Reply:
ill preserve the service. Not sure if it impacts the DNS address assignment.

Reply:
my bad i was thinking of pause unpause. For expiry i think we should delete all child resources.